tests/test_unit/test_roi/test_plot.py

```diff
@@ -0,0 +1,128 @@
+import re
+from typing import Any
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+from matplotlib.lines import Line2D
+from matplotlib.patches import PathPatch
+
+from movement.roi import LineOfInterest, PolygonOfInterest
+from movement.roi.base import BaseRegionOfInterest
+
+
+@pytest.fixture
+def havarti() -> PolygonOfInterest:
+    """Wedge-shaped RoI with several holes.
+
+    Havarti is a type of cheese that typically has holes and divots.
+    """
+    wedge = [(0, 0), (4, 0), (4, 0.5), (0, 2)]
+    hole1 = [(0.5, 0.5), (0.5, 0.75), (0.625, 0.625)]
+    hole2 = [(3, 0.25), (3.25, 0.5), (3.25, 0.75), (2.75, 0.75), (2.625, 0.5)]
+    hole3 = [(1.0, 0.1), (2, 0.1), (1.75, 0.2), (1.5, 0.3), (1.25, 0.2)]
+    hole4 = [(0.5, 1.75), (0.75, 1.5), (1.0, 1.25), (1.25, 1.25), (1.5, 1.4)]
+    return PolygonOfInterest(
+        wedge, holes=[hole1, hole2, hole3, hole4], name="Havarti"
+    )
+
+
+@pytest.fixture
+def octagonal_doughnut() -> PolygonOfInterest:
```

Comment:
Splitting hairs here, but if it's 18-sided, wouldn't it be "decaoctagonal"?

Reply:
(You might see this mistake in #440 too, but I have now done a CTRL+F to hopefully fix it :sweat_smile: )